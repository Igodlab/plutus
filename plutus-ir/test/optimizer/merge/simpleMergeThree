(let
  (nonrec)
  (termbind
    (vardecl x (con integer)) (con 1)
  )
  (let
    (nonrec)
    (termbind
      (vardecl y (con integer)) (con 2)
    )
    (let
      (nonrec)
      (termbind
        (vardecl z (con integer)) (con 3)
      )
      z
    )
  )
)
