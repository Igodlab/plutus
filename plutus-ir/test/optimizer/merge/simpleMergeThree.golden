(let
  (nonrec)
  (termbind (vardecl x (con integer)) (con 1))
  (termbind (vardecl y (con integer)) (con 2))
  (termbind (vardecl z (con integer)) (con 3))
  z
)